c/tskit/core.c

```diff
@@ -363,11 +357,20 @@ tsk_strerror_internal(int err)
             ret = "Bad genotype value provided";
             break;
 
-        /* Missing data errors */
+        /* Genotype decoding errors */
+        case TSK_ERR_TOO_MANY_ALLELES:
+            ret = "Cannot have more than 127 alleles";
```

Comment:
below, I think 127 is `INT16_MAX`; maybe this should be a sprintf with that value?

Reply:
It's either INT8_MAX or INT16_MAX, depending on whether we're using 16 bit genotypes. In practise, we'd need two error codes to distinguish these. Do you mean use sprintf so we're not hard coding the INT8_MAX? Quite awkward if so: these are statically allocated strings, and can't write to them without getting into parallelism issues etc.